03-monitoring-an-app/002-how-to-create-alarms.md

```diff
@@ -0,0 +1,99 @@
+---
+category: cloud-platform
+expires: 2019-07-01
+---
+# Creating your own custom alerts
+
+## Overview
+Alertmanager allows you define your own alert conditions based on [Prometheus expression language](https://prometheus.io/docs/prometheus/latest/querying/basics) expressions. 
+
+The aim of this document is to provide you with the necessary information to create and send application specific alerts to a Slack channel of your choosing.
+
+## Prerequisites
+This guide assumes the following:
+
+* You have [created a namespace for your application]({{ "/01-getting-started/002-env-create" | relative_url }})
+* Deployed at least one pod in your namespace 
+
+## Creating a slack webhook and amend Alertmanager
+This step requires the Cloud Platform team to create a receiver in [Alertmanager](https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/master/terraform/cloud-platform-components/templates/prometheus-operator.yaml.tpl#L115) and a [Slack webhook](https://api.slack.com/incoming-webhooks).
+
+Use the #ask-cloud-platform Slack channel to request a new alert route in Alertmanager. The team will need the following information:
+  
+- namespace name
+- team name
+- application name
+- slack channel
+
+The team will provide you with a `value` that'll need to be defined in the next step. Please copy it to your clipboard. 
+
+## Create a PrometheusRule
+A `PrometheusRule` is a custom resource that defines your triggered alert. This file will contain the alert name, promql expression and time of check. 
+
+To create your own custom alert you'll need to fill in the template below and deploy it to your namespace (tip: you can check rules in your namespace by running `kubectl get prometheusrule -n <namespace>`). 
+
+- Create a file called `prometheus-custom-rules-<application_name>.yaml`
+- Copy in the template below and replace the bracket values, specifying the requirements of your alert. The `<team_name>` field is the value you were passed earlier. 
```

Comment:
Same as above, I think it would be a bit more clear if we renamed `<team_name>` to `<custom_severity_level>` (using this name as an example).

Reply:
Changed in 
[15e81c5](/ministryofjustice/cloud-platform-user-docs/pull/114/commits/15e81c508f1a6b321666886927136e91d82f6925)